tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -421,6 +429,47 @@ func (a *AwsProvider) createInstance(imageID, instanceType, sshKey string,
 	return runResult.Instances[0], nil
 }
 
+// getLatestWindowsAMI returns the imageid of the latest released "Windows Server with Containers" image
+func (a *AwsProvider) getLatestWindowsAMI() (string, error) {
+	// Have to create these variables, as the below functions require pointers to them
+	windowsAMIOwner := "amazon"
+	windowsAMIFilterName := "name"
+	// This filter will grab all ami's that match the exact name. The '?' indicate any character will match.
+	// The ami's will have the name format: Windows_Server-2019-English-Full-ContainersLatest-2020.01.15
+	// so the question marks will match the date of creation
+	windowsAMIFilterValue := "Windows_Server-2019-English-Full-ContainersLatest-????.??.??"
```

Comment:
Do you want to make this as constant?

Reply:
I could not make it a constant and give the address of it